advanced/Scripts/query.sh

```diff
@@ -88,11 +61,41 @@ case "${options}" in
     *              ) domainQuery="${options}";;
```

Comment:
Suggest you use two different variable names for this process. I don't like modifying the variable and copying it back to itself when shell is the language. 

`*   ) rawDomainQuery="${options}";;`

`domainQuery=$(stuff with $rawDomainQuery)`

Reply:
Pushed